src/index.ts

```diff
@@ -18,7 +18,10 @@ type MessageTF =
 type BlobTF = ['blob', 'classic'] | ['blob', 'sha256'];
 type AddressTF = ['address', 'multiserver'];
 type EncryptionKeyTF = ['encryption-key', 'box2-dm-dh'];
-type IdentityTF = ['identity', 'po-box'] | ['identity', 'fusion'];
+type IdentityTF =
+  | ['identity', 'po-box']
+  | ['identity', 'group']
+  | ['identity', 'fusion'];
```

Comment:
Comment that does not block this PR: I just realized that ssb-bfe-spec doesn't have fusion, should we add it there?

Reply:
problem for another day lol